internal/bridge/bridge.go

```diff
@@ -9,6 +9,7 @@ import (
 var log = logrus.WithField("package", "bridge")
 
 type Bridge interface {
+	UpdateTicker(ticker storage.Ticker) error
```

Comment:
```suggestion
	Update(ticker storage.Ticker) error
```

I would only call it Update.

Reply:
This is changed now.